templates/taxi/driver_detail.html

```diff
@@ -3,6 +3,13 @@
 {% block content %}
   <h1>
     Username: {{ driver.username }}
+    <a href="{% url 'taxi:driver-delete' pk=driver.id %}" class="btn btn-danger link-to-page">
+      Delete
+    </a>
+
+    <a href="{% url 'taxi:driver-update' pk=driver.id %}" class="btn btn-secondary link-to-page">
+      Update
+    </a>
```

Comment:
Ensure that the URLs for deleting and updating the driver (`'taxi:driver-delete'` and `'taxi:driver-update'`) are correctly mapped to the intended views and that the view logic supports these operations.